api/src/main/java/org/openmrs/Obs.java

```diff
@@ -64,6 +64,32 @@
  */
 public class Obs extends BaseOpenmrsData {
 	
+	public enum Interpretation {
```

Comment:
Good point, I remembered for the getters/setters but I forgot it here. I will add this.